Random failure in test_child_raises_parent_exits_cleanly

```
___________ test_child_raises_parent_exits_cleanly[multiprocessing] ____________

backend = 'multiprocessing'

E         ValueError
E         
E         During handling of the above exception, another exception occurred:
E         
E         Traceback (most recent call last):
E           File "<string>", line 31, in <module>
E         NameError: name 'sleep' is not defined
E         
E       assert 1 == 0
E         +1
E         -0

```

observed in #1254:

https://dev.azure.com/joblib/joblib/_build/results?buildId=1803&view=logs&j=ee31ff9a-0e27-5056-2892-3ca864802f2f&t=75da501c-a363-5ac9-fd11-59ca96cb91ff

I will submit a PR.

--- a/joblib/test/test_memmapping.py
+++ b/joblib/test/test_memmapping.py
@@ -813,6 +813,7 @@ def test_child_raises_parent_exits_cleanly(backend):
     # - the resource_tracker does not emit any warnings.
     cmd = """if 1:
         import os
+        from time import sleep
 
         import numpy as np
         from joblib import Parallel, delayed

